rllib/utils/replay_buffers/prioritized_episode_replay_buffer.py

```diff
@@ -246,12 +248,13 @@ def sample(
         batch_length_T = batch_length_T or self.batch_length_T
 
         # Sample the n-step if necessary.
-        if n_step is None:
-            # If ' None' we sample from the closed interval (1, 5).
-            n_step = self.rng.integers(1, 5)
-        elif isinstance(n_step, tuple):
-            # Otherwise sample from the user-defined interval.
-            n_step = self.rng.integers(n_step[0], n_step[1])
+        _random_n_step = False
+        # If `n_step` is `None` we sample from the interval (1, 5).
+        n_step = n_step or (1, 5)
```

Comment:
No reason to do this check. Since the default should be a tuple, we can just provide this in the signature (it's immutable). But then again, should the default value for the arg `n_step` not simply be 1?

Reply:
I have set the default to 1.